### DISCUSSION TITLE
blog/modal-atoz/alert-dialog-focus-trap

### DISCUSSION THREAD

**Discussion by giscus** [Ssoon-m/ssoon.blog#16] [General]
# blog/modal-atoz/alert-dialog-focus-trap

포커스트랩 기능을 구현하고 리액트 컴포넌트로 깔끔하게 만드는 방법

https://ianlog.me/blog/modal-atoz/alert-dialog-focus-trap

**Comment by C17AN** [comment]
1 ~ 3편 모두 정말 재밌게 봤습니다.
좋은 글 감사드려요! 🙇‍♂️

**Reply by Ssoon-m** [reply]
감사합니다!!
